test/DockerSdk.Tests/Cli.cs

```diff
@@ -0,0 +1,111 @@
+        /// <returns>An array of lines written to stdout across all commands.</returns>
+        /// <remarks>
+        /// If any command returns a non-zero exit code or writes to stderr, the subsequent commands do not run. <br/>
+        /// Use of stdin is not supported.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">
+        /// One of the commands either wrote to stdout or returned a non-zero exit code.
+        /// </exception>
+        public static string[] Run(params string[] commands)
+            => Run(false, commands);
+
+        /// <summary>
+        /// Runs a series of commands in a Powershell prompt.
+        /// </summary>
+        /// <param name="ignoreErrors">True to ignore any errors that the commands may raise.</param>
+        /// <param name="commands">An array where each line is a command to run.</param>
+        /// <returns>An array of lines written to stdout across all commands.</returns>
+        /// <remarks>
+        /// If any command returns a non-zero exit code or writes to stderr, the subsequent commands do not run, unless
+        /// <paramref name="ignoreErrors"/> is true. <br/> Use of stdin is not supported.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">
+        /// One of the commands either wrote to stdout or returned a non-zero exit code, and <paramref
+        /// name="ignoreErrors"/> was false.
+        /// </exception>
+        public static string[] Run(bool ignoreErrors, params string[] commands)
+        {
+            var output = new List<string>();
+            foreach (var command in commands)
+                output.AddRange(Run(command, ignoreErrors));
+            return output.ToArray();
+        }
+
+        /// <summary>
+        /// Runs a command in a Powershell prompt.
+        /// </summary>
+        /// <param name="command">The command to run.</param>
+        /// <returns>An array of lines written to stdout.</returns>
+        /// <exception cref="InvalidOperationException">
+        /// The command either wrote to stdout or returned a non-zero exit code.
+        /// </exception>
+        public static string[] Run(string command)
+            => Run(command, false);
+
+        /// <summary>
+        /// Runs a command in a Powershell prompt.
+        /// </summary>
+        /// <param name="command">The command to run.</param>
+        /// <param name="ignoreErrors">True to ignore any errors that the command may raise.</param>
+        /// <returns>An array of lines written to stdout.</returns>
+        /// <exception cref="InvalidOperationException">
+        /// The command either wrote to stdout or returned a non-zero exit code, and <paramref name="ignoreErrors"/> is
+        /// false.
+        /// </exception>
+        public static string[] Run(string command, bool ignoreErrors)
+        {
+            var pi = new ProcessStartInfo("pwsh.exe", "-Command -")
```

Comment:
Haven't tested this yet, but would it be compatible with any OS?
Will check later what you are trying to do here

Reply:
Using PowerShell? Depending on the image that the build runner is using, yeah. It's [listed as available on Ubuntu 2004 runners](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md), for example.

That said, the current build failure is probably because of the .exe extension. I'll need to adjust that.

Reply:
OK, fixed. The tests that use this code all pass now.